test/e2e/rmv.js

```diff
@@ -0,0 +1,114 @@
+'use strict'
+
+const assert = require('assert')
+const tapePromise = require('tape-promise').default
+const tape = require('tape')
+
+const {createWhen} = require('./lib/util')
+const createClient = require('../..')
+const rmvProfile = require('../../p/rmv')
+const products = require('../../p/rmv/products')
+const createValidate = require('./lib/validate-fptf-with')
+const testJourneysStationToStation = require('./lib/journeys-station-to-station')
+const testRefreshJourney = require('./lib/refresh-journey')
+const testArrivals = require('./lib/arrivals')
+const testReachableFrom = require('./lib/reachable-from')
+
+const when = createWhen(rmvProfile.timezone, rmvProfile.locale)
+
+const cfg = {
+	when,
+	stationCoordsOptional: false,
+	products,
+	minLatitude: 47,
+	maxLatitude: 54,
+	minLongitude: 6,
+	maxLongitude: 11,
+}
+
+const validate = createValidate(cfg)
+
+const test = tapePromise(tape)
+const client = createClient(rmvProfile, 'public-transport/hafas-client:test')
+
+const frankfurtOstendstr = '3000525'
+const wiesbadenHbf = '3006907'
+
+test('journeys – Frankfurt Ostendstr. to Wiesbaden Hbf', async (t) => {
+	const res = await client.journeys(frankfurtOstendstr, wiesbadenHbf, {
+		results: 4,
+		departure: when,
+		stopovers: true
+	})
+
+	await testJourneysStationToStation({
+		test: t,
+		res,
+		validate,
+		fromId: frankfurtOstendstr,
+		toId: wiesbadenHbf
+	})
+	t.end()
+})
+
+// todo: via works – with detour
+// todo: without detour
+
+test('trip details', async (t) => {
+	const res = await client.journeys(frankfurtOstendstr, wiesbadenHbf, {
+		results: 1, departure: when
+	})
+
+	const p = res.journeys[0].legs[0]
+	t.ok(p.tripId, 'precondition failed')
+	t.ok(p.line.name, 'precondition failed')
+	const trip = await client.trip(p.tripId, p.line.name, {when})
+
+	validate(t, trip, 'trip', 'trip')
+	t.end()
+})
+
+test('arrivals at Wiesbaden Hbf', async (t) => {
+	const arrivals = await client.arrivals(wiesbadenHbf, {
+		duration: 10, when
+	})
+
+	validate(t, arrivals, 'arrivals', 'arrivals')
```

Comment:
Is there any reason you don't use the `testArrivals` method that's imported above here?

Reply:
`testArrivals` asserts that the resulting arrivals have the `stop.id`/`stop.station.id` that we passed in as `id`. Some endpoints, such as the RMV one, use varying stop IDs though.

Reply:
The test tooling is bad right now, will eventually™ improve it.